util/perl/TLSProxy/Proxy.pm

```diff
@@ -266,38 +309,6 @@ sub start
         die "no ACCEPT detected in '$execcmd' output: $?\n";
     }
 
-    # Just make sure everything else is simply printed [as separate lines].
-    # The sub process simply inherits our STD* and will keep consuming
-    # server's output and printing it as long as there is anything there,
-    # out of our way.
-    my $error;
-    $pid = undef;
-    if (eval { require Win32::Process; 1; }) {
-        if (Win32::Process::Create(my $h, $^X, "perl -ne print", 0, 0, ".")) {
-            $pid = $h->GetProcessID();
-            $self->{proc_handle} = $h;  # hold handle till next round [or exit]
-        } else {
-            $error = Win32::FormatMessage(Win32::GetLastError());
-        }
-    } else {
-        if (defined($pid = fork)) {
-            $pid or exec("$^X -ne print") or exit($!);
-        } else {
-            $error = $!;
-        }
-    }
-
-    # Change back to original stdin
-    open(STDIN, "<&", $savedin);
-    close($savedin);
-
-    if (!defined($pid)) {
-        kill(3, $self->{real_serverpid});
-        die "Failed to capture s_server's output: $error\n";
-    }
-
-    $self->{serverpid} = $pid;
-
```

Comment:
Why is this no longer needed?

Reply:
There's an explanation in the comments. I hope it answers this question.